src/components/home/Service.js

```diff
@@ -20,13 +20,11 @@ const Service = ({
       >
         {topic}
       </div>
-      <p
-        className={` ${textColor} flex text-center font-thin px-8 text-xl mt-4`}
-      >
+      <p className={` ${textColor} flex text-center px-8 text-xl mt-4`}>
         {description}
       </p>
       <div
-        className={`${mediaBG} flex justify-center text-center font-light text-4xl py-2 px-1 items-center w-6/12 h-full rounded-md mt-8`}
+        className={`${mediaBG} flex justify-center text-center font-bold text-4xl py-2 px-1 items-center w-6/12 h-full rounded-md mt-8`}
```

Comment:
also in service, can you change the media box to take up w-9/12 instead of 6/12. Then also add in whitespace-nowrap and min-w-fit in the className so the box does not get bigger like you have in the picture.